src/main/java/io/appium/java_client/HasDeviceDetails.java

```diff
@@ -0,0 +1,22 @@
+package io.appium.java_client;
+
+import static io.appium.java_client.MobileCommand.GET_DISPLAY_DENSITY;
+import static io.appium.java_client.MobileCommand.GET_SYSTEM_BARS;
+
+import java.util.Map;
+
+public interface HasDeviceDetails extends ExecutesMethod {
```

Comment:
It seems that methods that declared by this interface are Android-specific. Please take a look at 
https://github.com/search?q=org%3Aappium+DISPLAY+DENSITY&type=Code

https://github.com/search?q=org%3Aappium+GET+SYSTEM+BARS&type=Code

I think it has more sense to move this interface to the ```io.appium.java_client.android```